src/index.ts

```diff
@@ -16,16 +16,20 @@ import locationRoute from './routes/locationRoute';
 import familyRoute from './routes/familyRoute';
 import meaningRoute from './routes/meaningRoute';
 import cors from 'cors'; // Import the cors
```

Comment:
Might as well clean up this comment :D 
```suggestion
import cors from 'cors';
```